modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -136,7 +136,7 @@ import AposPublishMixin from 'Modules/@apostrophecms/ui/mixins/AposPublishMixin'
 import AposArchiveMixin from 'Modules/@apostrophecms/ui/mixins/AposArchiveMixin';
 import AposAdvisoryLockMixin from 'Modules/@apostrophecms/ui/mixins/AposAdvisoryLockMixin';
 import AposDocErrorsMixin from 'Modules/@apostrophecms/modal/mixins/AposDocErrorsMixin';
-import { detectDocChange } from 'Modules/@apostrophecms/schema/lib/detectChange';
+import * as detectChange from 'Modules/@apostrophecms/schema/lib/detectChange';
```

Comment:
TODO: find a way to keep the import as before, otherwise we'd need to change the way this lib is imported in doc-template-library and others...

<img width="1196" alt="image" src="https://github.com/user-attachments/assets/b066cc9a-9a28-440f-acfd-9321a6ee517f">

Reply:
cc @haroun if you've got an idea, since you've been working a lot with CommonJS/ESM recently

Reply:
issue is coming from https://github.com/apostrophecms/apostrophe/pull/4785/files#diff-430225e428cca721bb39ddb2fbdcff5c03ddbf95eaf8b73dbbfdeab9fc75537cR6-R36

All files in `ui/` must be ESM.